shared/charts/gpii-flowmanager/templates/virtual_service_cloudtrace.yaml

```diff
@@ -0,0 +1,21 @@
+{{- if .Values.enableStackdriverTrace }}
```

Comment:
`VirtualService` is not needed to allow access to external service, only the `ServiceEntry` -> remove

Reply:
Documentation says otherwise:
https://archive.istio.io/v1.0/docs/tasks/traffic-management/egress/#configuring-the-external-services

```Note that for TLS protocols, including HTTPS, the TLS VirtualService is required in addition to the ServiceEntry.```

Reply:
I double checked and everything seems to be working without `VirtualService` entries – outdated documentation?

Reply:
Yeah, that's my experience. `VirtualService` is typically required only if you want to affect how routing works (ie. split between 2 services or drop traffic or smth. like that).

Also, you're referencing some old version of docs (I know we're using 1.0), in current docs it's not there anymore: https://istio.io/docs/tasks/traffic-management/egress/egress-control/#access-an-external-https-service

Reply:
So it turns out that `VirtualService` is required after all (#448) :)